lib/active_force/sobject.rb

```diff
@@ -20,6 +20,8 @@ class SObject
     include ActiveModel::Serializers::JSON
     extend ActiveForce::Association
 
+    FIELD_OPTIONS = %i[as from sfdc_name]
```

Comment:
This should be frozen (`.freeze`) so it isn't accidentally mutated.  Also maybe it should be private?